### PR TITLE
[All Hosts] (manifest) simplify manifest validation instructions

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[707c759](https://github.com/OfficeDev/office-js-docs-pr/commits/707c7590d2ad33376477641f299bbaf19cf62a58)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-troubleshoot/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004170021547224-1757%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=THq7BXzgyonBP0ztqoPO%2F5HWtOpR37YD38uvZ1iSv0c%3D&st=2020-04-17T00%3A18%3A13Z&se=2020-05-18T00%3A23%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/troubleshoot-manifest.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ If you didn't use the [Yeoman generator for Office Add-ins](https://www.npmjs.co
...
     ```command&nbsp;line
-    npx office-addin-manifest validate MANIFEST_FILE
+    npm run validate
     ```
 
     > [!NOTE]
-    > If running this command results in the error message "The command syntax is not valid." (because the `validate` command is not recognized), run the following command to validate the manifest (replacing `MANIFEST_FILE` with the name of the manifest file): 
+    > If this command is not available or not working, run the following command instead to force the use of the latest version of the office-addin-manifest tool (replacing `MANIFEST_FILE` with the name of the manifest file):
     >
     > `npx --ignore-existing office-addin-manifest validate MANIFEST_FILE`
```
Comment (AlexJerabek) [inline docs/testing/troubleshoot-manifest.md:43]: Can we put this command in a code block? I think it'd be nice to have the copy button, but I don't know if that can be parsed inside a note.

```suggestion
    > ```command&nbsp;line
    > npx --ignore-existing office-addin-manifest validate MANIFEST_FILE
    > ```
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c5c817c](https://github.com/OfficeDev/office-js-docs-pr/commits/c5c817c5273e27b43f2d861f19e7e54364dd9308)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-troubleshoot/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004171633155245-1757%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FTJESeWbwlbVxp3a84psvzzojhWFYJRgJQUaYZdB8ew%3D&st=2020-04-17T16%3A29%3A38Z&se=2020-05-18T16%3A34%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/troubleshoot-manifest.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/testing/troubleshoot-manifest.md]: That worked. Thanks.

**Pull request merged**